web/src/components/Error.tsx

```diff
@@ -160,14 +174,7 @@ const ErrorComponent = (props: Props) => {
             <h4 className="list-group-item-heading"> Method</h4>
             {context.request_method}
           </ListGroup.Item>
-          <ListGroup.Item>
-            <h4 className="list-group-item-heading"> Headers</h4>
-            <Table striped>
-              <tbody>
-                { renderContextHeaders(context) }
-              </tbody>
-            </Table>
-          </ListGroup.Item>
+          {renderContextHeaders(context)}
```

Comment:
Alternatively, we could also conditionally call this function instead of possibly calling it with no headers. I'm thinking about something like `{ context.request_headers ? renderContextHeaders(context.request_headers) : null }`

Up to you!